shared/resources/SalesforceSDKResources.bundle/en.lproj/Localizable.strings

```diff
@@ -27,24 +27,27 @@
  */
 
 // Passcode view controller
-"nextScreenNavButtonTitle" = "Next";
-"prevScreenNavButtonTitle" = "Back";
-"createPasscodeNavTitle" = "Create Passcode";
-"confirmPasscodeNavTitle" = "Confirm Passcode";
-"verifyPasscodeNavTitle" = "Verify Passcode";
-"passcodeCreateInstructions" = "For increased security, please create a passcode that you will use to access the app when the session has timed out due to inactivity.";
-"passcodeChangeInstructions" = "For increased security, your admininstrator requires you to change your passcode.";
-"passcodeConfirmInstructions" = "Confirm the passcode you just entered.";
-"passcodeVerifyInstructions" = "Please enter your security passcode.";
-"minPasscodeLengthError" = "Your passcode must be at least %d characters long.";
-"passcodesDoNotMatchError" = "Passcodes do not match!";
-"passcodeInvalidError" = "The passcode you entered was invalid.";
-"forgotPasscodeTitle" = "Forgot Passcode?";
-"useTouchIdTitle" = "Use TouchId";
-"touchIdReason" = "Please authenticate to proceed.";
-"logoutAlertViewTitle" = "Are you sure you want to logout?";
-"logoutNo" = "No";
-"logoutYes" = "Yes";
+"createPasscodeNavTitle" = "Pin Code";
```

Comment:
There's some inconsistency with the casing of the word `Pin` throughout this file. Could we standardize on `PIN` (all uppercase)? It stands for `Personal Identification Number` anyway.

Reply:
I was just following the UX that was provided.  It is not inconsistent:  `Pin` is used in a nav bar title or button text when all words start with capitals, such as `Verify Pin Code`.  `pin` is used in normal instruction sentences, such as `Set up a pin code`.  

I would rather have `PIN` over `Pin/pin`, but then again I still think the correct answer is to follow Apple's example since they use `Passcode` for their strictly numeric device PIN.  

I can follow up with UX about this?

Reply:
I don't feel strongly about it. Either way is fine. If Apple uses "Passcode", maybe our UX should follow that pattern.